var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -103,7 +103,12 @@ def filter_compilers(self):
         nrnmech_makefile = join_path(self.prefix,
                                      "./bin/nrnmech_makefile")
 
-        assign_operator = "?="
+        # assign_operator is changed to fix wheel support
+        if self.spec.satisfies("+cmake") and self.spec.satisfies("@:7.99"):
+            assign_operator = "?="
+        else:
+            assign_operator = "="
```

Comment:
I don't see any `cmake` variant. So I think this conditional will always take the else branch and is equivalent to:
```suggestion
        assign_operator = "="
```

Reply:
LGTM

Reply:
@alalazo : thank's for catching that. I copied this from our fork where we still have autotools/cmake build.
I fixed the issue : With newer >=8.0 and develop we need this. Hence I just removed `+cmake` from the condition.